### DISCUSSION TITLE
18강 동영상 재생 관련 질문입니다.

### DISCUSSION THREAD

**Discussion by Qnanana** [SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course#243] [2024년도 교육 자료 관련 질문]
이전 질문으로 등록되었던 18번 동영상 재생과 관련하여 방법대로 했으나, 계속해서 진행이 되지 않아 여쭈어봅니다
혹시 몰라, VDL 미디어 파일을 설치하여 해보았습니다. 하지만, 다른 프로그램으로 열기를 클릭하면 영상 재생은 가능하지만, 18번 강의에서 진행하는 프로세스대로는 진행이 되지 않고 파일이 인식되지 않는다고 합니다. 
+ 동영상 연결일반 파일 내에서도 재생이 되지 않고, vs code 내에서도 인식이 되지 않습니다.
![스크린샷 2024-07-21 07-17-58](https://github.com/user-attachments/assets/4ce259d7-31bf-4af7-a7e8-c52655ed7329)
![스크린샷 2024-07-21 07-25-21](https://github.com/user-attachments/assets/63f72d67-d82f-43c5-b970-d8c6ca560356)
![스크린샷 2024-07-21 07-25-52](https://github.com/user-attachments/assets/ecc90d01-f496-41d5-93a9-cc7f74ba2d56)

**Comment by HyeongKeunHong** [comment]
18번 영상에서 vscode 상에서 영상을 확인하는 부분은 영상이 어떤 영상인지 대략적으로만 보여주는 부분입니다.
따라서 다른 프로그램으로 영상을 확인하셨다면, vscode상에서 확인을 하지 않으셔도 이후 과정 진행이 가능합니다.

**Reply by Qnanana** [reply]
해결되었습니다! 감사합니다!
